pom.xml

```diff
@@ -1306,26 +1316,26 @@
         <carbon.mediation.version>4.7.180</carbon.mediation.version>
 
         <!-- carbon identity -->
-        <carbon.identity.version>5.24.10</carbon.identity.version>
-        <carbon.identity.governance.version>1.7.2</carbon.identity.governance.version>
-        <carbon.identity.event.handler.account.lock.version>1.7.1</carbon.identity.event.handler.account.lock.version>
-        <carbon.identity.event.handler.notification.version>1.6.0</carbon.identity.event.handler.notification.version>
-        <carbon.identity-inbound-auth-oauth.version>6.9.8</carbon.identity-inbound-auth-oauth.version>
-        <carbon.identity-data-publisher-oauth.version>1.5.1</carbon.identity-data-publisher-oauth.version>
-        <carbon.identity-user-ws.version>5.6.1</carbon.identity-user-ws.version>
-        <carbon.identity-inbound-auth-openid.version>5.8.0</carbon.identity-inbound-auth-openid.version>
-        <carbon.identity-local-auth-basicauth.version>6.6.0</carbon.identity-local-auth-basicauth.version>
-        <carbon.identity-outbound-auth-samlsso.version>5.6.2</carbon.identity-outbound-auth-samlsso.version>
-        <carbon.identity-metadata-saml2.version>1.6.1</carbon.identity-metadata-saml2.version>
-        <carbon.identity-data-publisher-application-authentication.version>5.5.1</carbon.identity-data-publisher-application-authentication.version>
-        <carbon.extension.identity.oauth2.grantType.jwt.version>2.1.1</carbon.extension.identity.oauth2.grantType.jwt.version>
-        <carbon.identity.carbon.auth.saml2.version>5.7.2</carbon.identity.carbon.auth.saml2.version>
-        <carbon.identity.auth.version>1.7.1</carbon.identity.auth.version>
+        <carbon.identity.version>5.25.92</carbon.identity.version>
```

Comment:
5.25.684

Reply:
As mentioned in the description this PR is meant to upgrade the IS components to IS 6.1 compatible versions only. Component version upgrade to latest versions is addressed in https://github.com/wso2/product-apim/pull/13353.